flutter_todo/README.md

```diff
@@ -0,0 +1,12 @@
+# Realm Flutter Todo
+
+A todo list application built with the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/).
+
+This example only supports Flutter for Android, iOS, Windows, MacOS, and Linux.
+It **does not** support Flutter for Web.
+
+## Getting Started
+
+1. Clone and enter the project
```

Comment:
Should we provide the commands for this step. Perhaps it's not clear what enter the project means